lib/hand.rb

```diff
@@ -1,6 +1,16 @@
 require 'deck'
```

Comment:
We can actually get rid of this dependency since we're just accessing the Deck's constant